appveyor.yml

```diff
@@ -50,11 +50,11 @@ before_test:
   - sh: tail -f tests.log &
 
 test_script:
-  - mvn -V -B -U clean verify -DskipITs=false
+  - mvn -V -B -U clean verify -DskipITs=false  -Djna.nosys=true
```

Comment:
Why is "-Djna.nosys=true" needed?
Also there's a redundant white-space before "-Djna.nosys=true".

Reply:
We got a conflict on that package version while running the [tests](https://ci.appveyor.com/project/liron-shalom/jenkins-artifactory-plugin/builds/30691306).
The error message suggested that fix:

Reply:
Please reduce the double space to a single space.